frontends/concrete-python/concrete/fhe/compilation/compiler.py

```diff
@@ -456,6 +457,26 @@ def compile(
             )
         )
 
+        # if using tfhers integers, parameter selection strategy has to be multi-parameters.
+        # we try to catch this early, but the compiler will also fail and provide some info about
+        # it. So we don't need to scan all the inputset here.
+        if inputset is not None and len(inputset):  # type: ignore[arg-type]
```

Comment:
maybe a function check_input_set_has_tfhers_inputs, to keep it simple here and elsewhere